### DISCUSSION TITLE
How to obtain the number of parameters in a trained model?

### DISCUSSION THREAD

**Discussion by melodyL1** [pycaret/pycaret#3600] [Q&A]
How to obtain the number of parameters in a trained model?

![JH Z`G )23W0{ 2DSUKH)7G](https://github.com/pycaret/pycaret/assets/92028510/5ed9a2a2-e1ba-465e-a933-460e966e96cd)

**Comment by ngupta23** [comment]
Duplicate: Tracked here: https://github.com/pycaret/pycaret/issues/3601
